runelite-client/src/main/java/net/runelite/client/plugins/pestcontrol/PestControlPlugin.java

```diff
@@ -77,4 +87,21 @@ public void onChatMessage(ChatMessage chatMessage)
 			}
 		}
 	}
+
+	@Subscribe
+	public void onNpcSpawned(NpcSpawned event)
+	{
+		final NPC npc = event.getNpc();
+		if ("Spinner".equals(npc.getName()))
```

Comment:
Maybe you should use npc id instead

Reply:
This is simply mirroring the original code, but I'm fine with making that change.

Reply:
Made a set of the IDs and those are now checked rather than the NPC name.